fast follows for IDE lint/format page (#3500)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/dbt-cloud-ide/lint-format.md

3. In the code quality tool config pop up, you have the option to select **sqlfluff** or **sqlfmt**. 
4. To lint your code, select the **sqlfluff** radio button. (Use sqlfmt to [format](#format) your code)
5. Once you've selected the **sqlfluff** radio button, go back to the console section (below the **File editor**) to select the **Lint** or **Fix** dropdown button:

>     - **Lint button** &mdash; Displays linting issues in the IDE as wavy underlines in the **File editor**. You can hover over an underlined issue to display the details and actions, including a **Quick Fix** option to fix all or specific issues. After linting, you'll see a message confirming the outcome. Linting doesn't rerun after saving. Click on the **Lint** button again to rerun linting.

Reviewer: there's definitely conflicting style guidelines here. suggest just bolding just the UI label in this situation, instead of bolding everything to the left-hand side of the em dash: 

```suggestion
    - **Lint** button &mdash; Displays linting issues in the IDE as wavy underlines in the **File editor**. You can hover over an underlined issue to display the details and actions, including a **Quick Fix** option to fix all or specific issues. After linting, you'll see a message confirming the outcome. Linting doesn't rerun after saving. Click **Lint** again to rerun linting.
```

Author: yea i originally didnt highlight the button word but it looked weird so tried this, which appeared clearly, visually 😅

>     - **Fix button** &mdash; Auto-fixes errors in the **File editor**. Once you've auto-fixed, you'll see a message confirming the outcome. 

Reviewer: ```suggestion
    - **Fix** button &mdash; Automatically fixes linting errors in the **File editor**. When fixing is complete, you'll see a message confirming the outcome. 
```

<Lightbox src="/img/docs/dbt-cloud/cloud-ide/python-black.gif" width="95%" title="Format Python files using Black."/>

## FAQs

<details>
<summary>Can I nest <code>.sqlfluff</code> files?</summary>

To ensure optimal code quality, consistent code, and styles &mdash; it's highly recommended you have one main `.sqlfluff` configuration file in the root folder of your project. Having multiple files can result in various different SQL styles in your project. <br /><br />

However, you can customize and include an additional child `.sqlfluff` configuration file within specific subfolders of your dbt project. <br /><br />By nesting a `.sqlfluff` file in a subfolder, SQLFluff will apply the rules defined in that subfolder's configuration file to any files located within it. The rules specified in the parent `.sqlfluff` file will be used for all other files and folders outside of the subfolder. This hierarchical approach allows for tailored linting rules while maintaining consistency throughout your project. Refer to [SQLFluff documentation](https://docs.sqlfluff.com/en/stable/configuration.html#configuration-files) for more info.

</details>

<details>

> <summary>Can I run SQLFluff commands form the terminal?</summary>

Reviewer: ```suggestion
<summary>Can I run SQLFluff commands from the terminal?</summary>
```

> Running SQLFluff commands from the terminal isn't supported, but it's currently on the Product team's roadmap for future support.

Reviewer: i've been taught to avoid adding this kind of info in the user docs for these reasons:
- it can be seen as a promise we're making and could be a liability for us (especially if the roadmap changes)
- it can create support churn when people reach out asking when the functionality will be available.

```suggestion
Currently, running SQLFluff commands from the terminal isn't supported. 
```

Author: yep usually i wouldn't add this for the exact same reasons but i think this was fine per pm. 

i'd like to get this merged asap so I'll take the suggestion to get this live.
